drivers/bus/pci/linux/pci_uio.c

```diff
@@ -282,7 +286,20 @@ pci_uio_alloc_resource(struct rte_pci_device *dev,
 			goto error;
 		}
 	}
+#endif
 
+#ifdef COMPOSITE_EXISTS
+	RTE_SET_USED(pci_uio_set_bus_master);
+	RTE_SET_USED(loc);
+	RTE_SET_USED(uio_cfg_fd);
+	RTE_SET_USED(fd);
+	RTE_SET_USED(uio_num);
+	RTE_SET_USED(cfgname);
+	RTE_SET_USED(dirname);
```

Comment:
I don't really like this. But seems this is the only way. Do you think if this will change if we move to different platforms which have different PCI devices？

Reply:
This is just to remove compiling warnings, why will this affect different PCI devices? I don't think this is a key point.